test/unit/api/events.spec.ts

```diff
@@ -23,6 +17,47 @@ describe('Event Tests', () => {
       .get('/api/getRewards')
       .set('Accept', 'application/json')
       .query({ chain });
-    console.log(res.body);
+    expect(res.body.status).to.be.equal('Success');
+    expect(res.body.validators).to.not.be.null;
+  });
+  it('should fetch global statistics from HistoricalValidator and Validator model', async () => {
+    chai.request(app)
+      .get('/api/getGlobalStatistics')
+      .set('Accept', 'application/json')
+      .then((res) => {
+        expect(res.body.status).to.equal('Success');
+        expect(res.body.result).to.not.be.null;
+      })
+      .catch(error => {
+        console.log(error);
+      });
+
+  });
+
+  it('should fetch current validator details database', async () => {
```

Comment:
same as above, we're using async/await not promise syntax.

Reply:
Noted